src/views/SignDidExtrinsic/SignDidExtrinsic.tsx

```diff
@@ -326,29 +331,31 @@ export function SignDidExtrinsic({ identity }: Props): JSX.Element | null {
     return null; // blockchain data pending
   }
 
+  const identityIsPredetermined = did && did === didUri;
+  const Carousel = identityIsPredetermined ? IdentitySlide : IdentitiesCarousel;
```

Comment:
I thought `Identity` could be a better name since its only a carousel in the 2nd case, but I guess that could be 
mixed up with the `Identity` interface.

What about `CurrentIdentity`? 🤔

Reply:
I’m unhappy with both options, but I came up with a third. How do you like `IdentityChoices`? Sometimes you have a choice of several, sometimes of one :shrug:

Reply:
I like `IdentityChoice` even better, since choice can imply both the chosen one or the act of choosing between several.